ichnaea/webapp/app.py

```diff
@@ -51,20 +49,37 @@ def worker_exit(server, worker):
     shutdown_worker(_APP)
 
 
-def log_access_factory(wsgi_app):
-    """WSGI middleware for logging HTTP requests."""
+def log_middleware(wsgi_app):
```

Comment:
`log_middleware` felt like a better name than `log_access_factory`, and the class comment seemed to agree.